vcd/resource_vcd_independent_disk_test.go

```diff
@@ -42,11 +49,34 @@ func TestAccVcdIndependentDiskBasic(t *testing.T) {
 		CheckDestroy: testDiskResourcesDestroyed,
 		Steps: []resource.TestStep{
 			resource.TestStep{
-				Config: configText,
+				Config: configTextForCompatibility,
 				Check: resource.ComposeTestCheckFunc(
-					testAccCheckDiskCreated("vcd_independent_disk." + resourceName),
+					testAccCheckDiskCreated("vcd_independent_disk."+resourceName),
+					//resource.TestCheckResourceAttr("vcd_independent_disk."+resourceName, "size_in_bytes", "5242880000"),
+					resource.TestMatchResourceAttr("vcd_independent_disk."+resourceName, "owner_name", regexp.MustCompile(`^\S+`)),
+					resource.TestMatchResourceAttr("vcd_independent_disk."+resourceName, "datastore_name", regexp.MustCompile(`^\S+`)),
+					resource.TestMatchResourceAttr("vcd_independent_disk."+resourceName, "iops", regexp.MustCompile(`^\d+$`)),
+					resource.TestCheckResourceAttr("vcd_independent_disk."+resourceName, "is_attached", "false"),
 				),
 			},
+			/*			resource.TestStep{
+						Config: configText,
+						Check: resource.ComposeTestCheckFunc(
+							testAccCheckDiskCreated("vcd_independent_disk."+resourceName+"second"),
+							resource.TestCheckResourceAttr("vcd_independent_disk."+resourceName+"second", "size_in_bytes", "5242880000"),
+							resource.TestMatchResourceAttr("vcd_independent_disk."+resourceName+"second", "owner_name", regexp.MustCompile(`^\S+`)),
+							resource.TestMatchResourceAttr("vcd_independent_disk."+resourceName+"second", "datastore_name", regexp.MustCompile(`^\S+`)),
+							resource.TestMatchResourceAttr("vcd_independent_disk."+resourceName+"second", "iops", regexp.MustCompile(`^\d+$`)),
+							resource.TestCheckResourceAttr("vcd_independent_disk."+resourceName+"second", "is_attached", "false"),
```

Comment:
It looks to me we should be testing all these fields, only the size_in_bytes should be commented out.

Reply:
This part was specific for back compatibility. Other test part is normal.